### ISSUE TITLE
“Copy settings to other side” ignores “advanced” expand/collapsed state

### ISSUE THREAD

**Issue by surma** [GoogleChromeLabs/squoosh#1032]
1. Open image
2. Select AVIF (or any other codec with “advanced options”
3. Expand advanced options and change some of them
4. Click “copy settings to other side”
5. Advanced section on other side is _not_ expanded (but settings are indeed copied)

**Comment by jakearchibald** [comment]
Yeah, right now we copy over the codec settings, not the UI state.

It might make more sense to use UI state throughout the app, and only convert to codec settings at the last moment (in the worker).
